docs/sources/alerting/fundamentals/contact-points/index.md

```diff
@@ -47,7 +47,7 @@ The following table lists the contact point integrations supported by Grafana.
 | [Google Chat](https://chat.google.com/)          | `googlechat`              | Supported            | N/A                                                                                                      |
 | [Kafka](https://kafka.apache.org/)               | `kafka`                   | Supported            | N/A                                                                                                      |
 | [Line](https://line.me/en/)                      | `line`                    | Supported            | N/A                                                                                                      |
-| [Microsoft Teams](https://teams.microsoft.com/)  | `teams`                   | Supported            | N/A                                                                                                      |
+| [Microsoft Teams](https://teams.microsoft.com/)  | `teams`                   | Supported            | Supported                                                                                                |
```

Comment:
This is the only place I could find in the docs where external AM integrations are listed. If there are any other places, happy to update as needed!